tensorflow/lite/micro/micro_interpreter_test.cc

```diff
@@ -317,8 +334,9 @@ TF_LITE_MICRO_TEST(TestIncompleteInitializationAllocationsWithSmallArena) {
   // Interpreter fails because arena is too small:
   TF_LITE_MICRO_EXPECT_EQ(interpreter.Invoke(), kTfLiteError);
 
-  TF_LITE_MICRO_EXPECT_EQ(
-      static_cast<size_t>(192),
+  // The head buffer use cannot exceed the upper bound from x86.
+  TF_LITE_MICRO_EXPECT_GE(
```

Comment:
+1

The guidance is for code to read just like english. Since the comment reads head buffer less than max_size, it suggests that the code should also be written in that way.

http://go/c-readability-advice#avoid-yoda-style-comparisons

Reply:
Done. Thx for the suggestions!

Agree with Nat on GetHeadUserdBytes() as well. For now, keep it as it is and look for ways in future to improve it.